engine/src/main/java/org/hibernate/search/engine/mapper/mapping/building/impl/ConfiguredIndexSchemaNestingContext.java

```diff
@@ -91,6 +93,11 @@ public String toString() {
 			while ( nextDotIndex >= 0 ) {
 				String objectName = prefixToParse.substring( afterPreviousDotIndex, nextDotIndex );
 				contextBuilder.appendObject( objectName );
+
+				// Make sure to mark the paths as encountered in the filter
+				String objectNameRelativeToFilter = prefixToParse.substring( 0, nextDotIndex );
+				filter.isPathIncluded( objectNameRelativeToFilter );
```

Comment:
I would use the name `markPathIncluded` or `markPathAsIncluded`.

Reply:
The thing is, this method returns a boolean telling you whether the path is included or not. Here's I'm just twisting it: I know it has a side effect, which is marking the path as encountered, so I use it just for the side effect.

Every where else we really mainly use the method to know whether the path is included or not.

I'll add a comment to clarify.